docs/roadmap.md

```diff
@@ -4,25 +4,27 @@
 
 **yunikorn-core**
 
-* Gang Scheduling.
-* Priority based preemption.
-* Support application priority.
-* Rich scheduler metrics.
-* Workload simulator.
-* Prometheus integration (phase 2).
-* Grafana integration.
+* [YUNIKORN-1](https://issues.apache.org/jira/browse/YUNIKORN-1): Support app/task priority aware scheduling.
+* [YUNIKORN-2](https://issues.apache.org/jira/browse/YUNIKORN-2): Gang Scheduling.
+* [YUNIKORN-21](https://issues.apache.org/jira/browse/YUNIKORN-21): Optimize node sorting algorithms.
+* [YUNIKORN-42](https://issues.apache.org/jira/browse/YUNIKORN-42): High efficient scheduling events framework phase 1.
+* [YUNIKORN-33](https://issues.apache.org/jira/browse/YUNIKORN-33): Performance benchmark with Kubemark.
+* [YUNIKORN-131](https://issues.apache.org/jira/browse/YUNIKORN-131): Prometheus integration - phase 2.
+* [YUNIKORN-132](https://issues.apache.org/jira/browse/YUNIKORN-132): Publish helm charts for convenient access.
```

Comment:
Do we need this? 
Is there any public helm repo in apache where we can publish same?

Reply:
No, I don't think apache has that.
If you are not comfortable with this, I am fine to remove it.

Reply:
the helm chart could be deployed and accessible from github by using `chart-releaser` the index.yaml file we can even host on the standard yunikorn website but this is a lot of work and would need good amount of thought, planning and build changes too.

I would prefer to not do this for now

Reply:
OK, let me remove it.

Reply:
Ok.